### DISCUSSION TITLE
Unsure how to get the correct version of the mod

### DISCUSSION THREAD

**Discussion by LaCod6** [Squidpad/BG3DevouringAndDigesting#29] [Q&A]
I got the mod to work find, but it is the version that was downloadable from mod nexus. I used BGMM not vortex, so everything is working find. However, I noticed that there is a fix for burping cantrip. The BGMM seems to not recognize the git download version as a mod because of its lack of a .pak file. 

Now I am not great a these sorts of things, so I am unclear on how to get that fix. Should I just take that file itself and add it to the mod nexus file download before importing it to the BGMM? Any help is appreciated!

**Comment by Squidpad** [comment]
On the front page of this Github, there's a "Releases" section. Download "DevouringAndDigesting.zip" from the "Assets" section there.
![Screenshot_20241103_124008-1](https://github.com/user-attachments/assets/dcefa699-a50a-4335-9d1c-119466d793aa)

**Reply by LaCod6** [reply]
Thank you!
